### DISCUSSION TITLE
engine='xtts', xtts=None ...Am I missing a setting?

### DISCUSSION THREAD

**Discussion by Nikanoru** [aedocw/epub2tts#198] [Q&A] [answered]
I just noticed the "xtts=None" in the terminal output after executing the command 

epub2tts AN.epub --start 4 --end 4 --engine xtts --speaker "Nova Hogarth"

It all sounds fine to me, but that xtts=None does make me wonder if I am missing a setting that would further improve the TTS quality.

![image](https://github.com/aedocw/epub2tts/assets/83954293/f9d94ce8-121c-44f1-a341-b5774a59be98)

**Comment by aedocw** [comment, marked as answer]
Nope, you are not missing anything. The reason for that is basically backwards compatibility. Before Coqui released their studio speakers, the only way to use XTTS was by supplying at least one sample to use for voice cloning. I probably should have made the usage be "--engine xtts --samples [sample.wav,sample2.wav,etc]"... That would have kept things consistent, but I didn't expect the change that happened.

Because I didn't want to introduce a change that would break previous usage, once studio speakers became an option I thought it best to use the "--engine xtts --speaker <speaker>" approach so there would be a new and explicit way to choose the studio speakers.
